grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/gradle/MigrationCriteria.kt

```diff
@@ -96,7 +94,7 @@ internal class DependenciesMigrationCriteria @Inject constructor(
     override fun canMigrate(project: Project): Boolean {
         val hasPrivateDependencies =
             dependenciesDataSource.hasDepsFromUnsupportedRepositories(project)
-        val hasIgnoredArtifacts = dependenciesDataSource.hasIgnoredArtifacts(project)
+        val hasIgnoredArtifacts = false // TODO(arun) Refactor
```

Comment:
Should this be `@Deprecated`?

Reply:
Yes, will mark it as such thanks. We no longer need that check.